src/main.rs

```diff
@@ -266,6 +266,26 @@ fn remove_dependencies(manifest: &str, dependencies_list: &[String]) -> anyhow::
             .with_context(|| format!("Dependency {k} not found"))?;
     }
 
+    let features = manifest
+        .iter_mut()
+        .find_map(|(k, v)| (v.is_table_like() && k == "features").then_some(Some(v)))
+        .flatten()
+        .and_then(|v| v.as_table_mut());
+
+    if let Some(features) = features {
+        for (_, deps) in features.iter_mut() {
+            if let Some(deps) = deps.as_array_mut() {
+                deps.retain(|dep| {
+                    if let Some(dep) = dep.as_str() {
+                        !dependencies_list.iter().any(|d| dep.contains(d))
```

Comment:
I think `contains` is a bit too vague here: a dependency flagged for removal called `log` would cause removal of a feature enabling *something* (a feature or another dependency) called `flog`.

Can you handle that better, and add a test that checks that too, please?